packages/api/endpoints/token.js

```diff
@@ -204,6 +179,49 @@ async function login(request, oAuth2Provider) {
   return buildPermanentRedirectResponse(authorizationUrl);
 }
 
+/**
+ * Handle token deletion
+ *
+ * @param {Object} request - an API Gateway request
+ * @returns {Object} an API Gateway response
+ */
+async function deleteToken(request) {
+  const body = request.body
+    ? JSON.parse(request.body)
+    : {};
+  const requestJwtToken = get(body, 'token');
+
+  if (!requestJwtToken) {
+    return new MissingTokenResponse();
```

Comment:
Do we really need to send back an error on this? If they just want to log out from the dashboard then does it matter?
As long as after this action there is/remains no existing token?

Reply:
Hmm, that's a good question. I think we may still want to validate the request so that in case the dashboard code (or any client) was not sending valid requests to delete their tokens, then they would be informed of that. Otherwise their tokens wouldn't be getting deleted from the API but they would have no way of recognizing that. What do you think?

Reply:
What would be the cases that this doesn't work? 

I'm just thinking about a user who tries to log off then getting an error and them getting upset. 
If there is a missing token, then the job is already done? and we shouldn't worry the user about it.

Reply:
This response is hit when the token is missing from the request itself, not from the database. So if this line is hit, then the API cannot attempt to delete the record from the database.

For the dashboard, it should always work, since we wrote the code integrating with the endpoint. However if for some unforeseen reason it didn't work, then all that would happen is a failed request logged to the console, so the failure would be silent from the user perspective. But if it were failing because somehow the code was regressed and was not including a token parameter, I feel like having the error clearly identified in the response would be helpful.

Also, while unlikely, it is possible that other clients could use this API and it seems helpful to validate the request and return helpful error responses